script.js

```diff
@@ -0,0 +1,100 @@
+const hamburger = document.getElementById('menuBtn');
+const quitBtn = document.querySelector('.btnClose');
+const menuPopup = document.getElementById('menu-bar');
+const menuItems = document.querySelectorAll('.menu-item');
+
+menuPopup.classList.add('hidden');
+
+function closePopup() {
+  hamburger.classList.toggle('hidden');
+  quitBtn.classList.toggle('hidden');
+  menuItems.forEach((item) => { menuPopup.remove(item); });
+  menuPopup.classList.toggle('hidden');
+}
```

Comment:
- [ ] It seems like there is a bug with your `closePopup` function. When I click for the first time, I can see the menu being displayed properly, but after closing it and trying to open it again, it is not working and the menu icon disappears from the screen.

Reply:
@Lordkaito Everything else is fixed. I'm still dealing with that slippery bug... any clue is welcome 👾 😏

Reply:
- [ ] The bug is still not solved. After closing the menu, the user is not able to open menu again. Kindly work on that.